### PR TITLE
KEYCLOAK-13445 Modify SigningInTest for changes in credential type

### PR DESCRIPTION
**Pull request by Aboullos**
Modify SigningIn tests according to the functionality changes in the credential types

### CODE REVIEW

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java [hunk 1]
```diff
@@ -240,6 +240,9 @@ public void otpTest() {
         assertEquals(2, otpCredentialType.getUserCredentialsCount());
         assertUserCredential(label2, true, otp2);
 
+        assertFalse("Not set up link is visible", otpCredentialType.isNotSetUpLinkVisible());
+        assertTrue("Title is not visible", otpCredentialType.isTitleVisible());
```
Comment (vmuzikar) [inline testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java]: Before this, we need to disable given required action and also add an assert to check that the setup link is not visible. We could also check the user credential count didn't change after that. Something like:
```suggestion
        assertTrue("Set up link is not visible", otpCredentialType.isSetUpLinkVisible());

        RequiredActionProviderRepresentation requiredAction = new RequiredActionProviderRepresentation();
        requiredAction.setEnabled(false);
        testRealmResource().flows().updateRequiredAction(CONFIGURE_TOTP.name(), requiredAction);

        assertFalse("Set up link is visible", otpCredentialType.isSetUpLinkVisible());
        assertFalse("Not set up link is visible", otpCredentialType.isNotSetUpLinkVisible());
        assertTrue("Title is not visible", otpCredentialType.isTitleVisible());
        assertEquals(2, otpCredentialType.getUserCredentialsCount());
```

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java [hunk 2]
```diff
@@ -286,6 +289,9 @@ private void testWebAuthn(boolean passwordless) {
         assertEquals(2, credentialType.getUserCredentialsCount());
         assertUserCredential(label2, true, webAuthn2);
 
+        assertFalse("Not set up link is visible", credentialType.isNotSetUpLinkVisible());
+        assertTrue("Title is visible", credentialType.isTitleVisible());
```
Comment (vmuzikar) [inline testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java]: Same as above in `otpTest`.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@Aboullos Thank you for the fix! Please see comments below.

**Review by vmuzikar — APPROVED** [review]
LGTM, thank you @Aboullos!

**Review by hmlnarik — APPROVED** [review]

**Comment by hmlnarik** [comment]
Travis run - https://travis-ci.com/github/keycloak/keycloak/builds/155045890

**Review by pdrozd — APPROVED** [review]

**Comment by hmlnarik** [comment]
Travis failed to report back for some reason but the tests passed. Thus merging regardless

**Pull request merged**